client/src/Model/MainModel.ts

```diff
@@ -0,0 +1,36 @@
+import { dummyhistories } from '@/assets/dummy';
+import Observable from '@/Core/Observable';
+import { IHistory } from '@/utils/types';
+
+class MainModel extends Observable {
+  private historyCards: IHistory[];
+
```

Comment:
지금은 일단 모델 자체에 `key` 값을 두고 활용하면 좋을 거 같습니다~~

```suggestion
  key: string = 'history';
```